lib/private/SystemTag/SystemTagManager.php

```diff
@@ -206,6 +208,11 @@ public function updateTag(
 			);
 		}
 
+		$user = $this->userSession->getUser();
+		if (!$this->canUserCreateTag($user)) {
```

Comment:
Even if it's an alias, probably better to use it:
```suggestion
		if (!$this->canUserUpdateTag($user)) {
```